superset-frontend/plugins/plugin-chart-echarts/src/utils/helper.ts

```
@@ -0,0 +1,5 @@
+export const convertNumber = (value: string | number) => {
+  /* eslint radix: ["error", "as-needed"] */
```

1 more nit - maybe we could just pass radix `10` instead of ignoring the error?

---

I was wondering the same. I think there's _some_ chance of it parsing some implicit octal radix, but I haven't stopped to try to break it.

---

Resolved with radix 10